tests/test_command_routes.py

```
@@ -0,0 +1,313 @@
+    "teslaAccessToken": "at_test",
+    "teslaRefreshToken": "rt_test",
+    "teslaHttpProxy": "http://localhost:4443",
+    "teslaClientId": "cid_test",
+    "chargerMaxAmps": 32.0,
+    "chargerMinAmps": 6.0,
+    "priority": 1,
+    "enabled": True,
+}
+
+USERNAME = "tsc-admin"
+PASSWORD = "correct-horse-battery"
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to: "PASSWORD"

(S105)